packages/elements/src/Text/models/Text/index.ts

```diff
@@ -24,6 +25,9 @@ export const getTextModel = (data: Data): ElementModel => {
   // Transform all inside div to P
   node = transformDivsToParagraphs(node);
 
+  // Wrap all single spans from the first level in paragraph
+  node = wrapSingleSpansInParagraph(node);
+
```

Comment:
This is not necessary because the browser will automatically fix the issue by closing the first <p> tag before the second one starts, as this nesting is invalid in HTML.